server/pulp/server/webservices/views/serializers/link.py

```diff
@@ -0,0 +1,59 @@
+"""
+Generation of link objects for REST object serialization.
+
+link object:
+{
+    "_href": <uri path to resource or collection>
+}
+"""
+
+import copy
+
+from pulp.server.webservices import http
+
+
+_LINK_OBJ_SKEL = {
+    '_href': None,
+}
+
+
+def link_obj(href):
+    """
+    Create a link object for an arbitrary path.
+    @param href: uri path
+    @type  href: str
+    @return: link object
+    @rtype:  dict
+    """
+    link = copy.copy(_LINK_OBJ_SKEL)
+    link['_href'] = href
+    return link
+
+
+def current_link_obj():
+    """
+    Create a link object for the path for the current request.
+    @return: link object
+    @rtype:  dict
+    """
+    link = copy.copy(_LINK_OBJ_SKEL)
+    link['_href'] = http.uri_path()
+    return link
+
+
+def child_link_obj(*path_elements):
+    """
+    Create a link object that appends the given elements to the path of the
+    current request.
+    Example: current request path = '/foo/bar/baz/'
+             path elements = ['fee', 'fie']
+             returned path = '/foo/bar/baz/fee/fie/'
+    @param path_elements: path elements to append
```

Comment:
s/path elements to append/Captured positional arguments treated as path elements to append/